mne/defaults.py

```diff
@@ -60,6 +64,10 @@
         fnirs_fd_ac_amplitude="V",
         fnirs_fd_phase="rad",
         fnirs_od="V",
+        fnirs_td_gated_amplitude="AU",  # counts
+        fnirs_td_moments_intensity="AU",  # counts
+        fnirs_td_moments_mean="s",
+        fnirs_td_moments_variance="s²",
```

Comment:
@julien-dubois-k better? And in particular the `S` you suggested for the unit is for Siemens?

Reply:
yes these are good. `s` is seconds

Reply:
hah, whoops! Will fix